help/en/html/doc/Technical/JVMCapabilities.shtml

```diff
@@ -832,6 +832,12 @@
             number of "access violation" messages.  These are just advisory
             with Java 17, but will become mandatory failures in some later
             version of Java (Java 21?)
+        <li>(Development only) Improved error checking in the Javadoc
+            process generates hundreds of warnings that can hide real problems.
+            These will need to be cleaned up.
+        <li>(Development only) Deprecations that are marked for removal 
+            generate un-suppressible warning messages.
+            These will need to be cleaned up.
```

Comment:
Thanks for the heads up. Can you attach the output of those warnings, both javadoc and deprecations?

I'm mostly interested in if there are issues with LogixNG that I need or should take care of.